pokemongo_bot/__init__.py

```diff
@@ -396,18 +427,14 @@ def _set_starting_position(self):
             return
 
         if self.config.location:
-            try:
-                location_str = self.config.location.encode('utf-8')
-                location = (self._get_pos_by_name(location_str.replace(" ", "")))
-                self.api.set_position(*location)
-                logger.log('')
-                logger.log(u'Location Found: {}'.format(self.config.location))
-                logger.log('GeoPosition: {}'.format(self.position))
-                logger.log('')
-                has_position = True
-            except Exception:
```

Comment:
It was hiding geolocation errors that shouldn't be hidden, like request denied. But I committed in this PR by mistake. Undo or keep these changes?